meet-data/apa/1711/entries.csv

```diff
@@ -19,13 +19,13 @@ Ken Morgan,MS,SBD,M,Master I.,Wraps,124.38,125,42,306.17,213.19,292.57,811.93,1
 Bubba Norris,MS,SBD,M,Submaster,Wraps,86.09,90,33,174.63,111.13,224.53,510.29,1
 Chris Pereira,MS,SBD,M,Open,Wraps,88.9,90,20,258.55,167.83,265.35,691.73,1
 Jake Shelton,MS,SBD,M,Open,Wraps,139.34,140,22,299.37,183.7,292.57,775.64,1
-Cameron Moss,AL,SBD,M,Junior (by Formula),Wraps,117.66,125,20,276.69,199.58,331.12,807.39,1
-Damien Seales,MS,SBD,M,Junior (by Formula),Wraps,65.14,67.5,20,208.65,113.4,247.21,569.26,2
-Chris Pereira,MS,SBD,M,Junior (by Formula),Wraps,88.9,90,20,258.55,167.83,265.35,691.73,3
-Jake Shelton,MS,SBD,M,Junior (by Formula),Wraps,139.34,140,22,299.37,183.7,292.57,775.64,4
-Tony Beckham,MS,SBD,M,Junior (by Formula),Wraps,60.06,60,20,142.88,120.2,174.63,437.72,5
-Joshua Hayes,FL,SBD,M,Junior (by Formula),Wraps,99.34,100,20,215.46,138.35,240.4,594.21,6
-Eric Elsey,MS,SBD,M,Junior (by Formula),Wraps,69.31,75,21,,,,,DQ
+Cameron Moss,AL,SBD,M,Junior,Wraps,117.66,125,20,276.69,199.58,331.12,807.39,1
+Damien Seales,MS,SBD,M,Junior,Wraps,65.14,67.5,20,208.65,113.4,247.21,569.26,2
+Chris Pereira,MS,SBD,M,Junior,Wraps,88.9,90,20,258.55,167.83,265.35,691.73,3
+Jake Shelton,MS,SBD,M,Junior,Wraps,139.34,140,22,299.37,183.7,292.57,775.64,4
+Tony Beckham,MS,SBD,M,Junior,Wraps,60.06,67.5,20,142.88,120.2,174.63,437.72,5
```

Comment:
This one should just have bodyweight changed to 60 -- they meant to enter him as "maximum allowable by weightclass" but then got caught by kg/lbs rounding.